rosetta/text/text_processors.py

```diff
@@ -659,10 +660,11 @@ def set_bit_precision_required(self):
 
     def filter_sfile(
         self, infile, outfile, doc_id_list=None, enforce_all_doc_id=True,
```

Comment:
FYI:  I messed up by using `doc_id_list`..this specifies that it should be a list, but any old iterable will do.  So I should have called it `doc_ids` and said it was an "iterable over strings, representing doc ids".  No sense in changing it since that will change the API, but in the future I should be more careful.